src/lib/repository.js

```diff
@@ -23,7 +23,7 @@ function getRemoteUrl (pkg, callback) {
   }
   if (/^git\+/.test(pkg.repository.url)) {
     pkg.repository.url = pkg.repository.url.substr(4)
-    pkg.repository.url = pkg.repository.url.replace(/^ssh:\/\/git@/, 'https://')
+    pkg.repository.url = pkg.repository.url.replace(/^(ssh:\/\/git|git:\/\/)@/, 'https://')
```

Comment:
Thank you for the fix. I think the `@` should be a part from the `ssh://git@` url scheme? Then the regexp should look like `^(ssh:\/\/git@|git:\/\/)`.